example-library/src/main/java/uk/gov/gchq/palisade/example/library/rule/PassThroughRule.java

```diff
@@ -27,4 +27,9 @@
     public Resource apply(final Resource resource, final User user, final Context context) {
         return resource;
     }
+
+    @Override
+    public boolean isApplicable( final User user, final Context context) {
```

Comment:
SonarQube has picked you up on this
```suggestion
    public boolean isApplicable(final User user, final Context context) {
```